site/src/pages/components/invokers.explainer.mdx

```diff
@@ -357,6 +357,119 @@ will be `aria-expanded=true` when the _Invokee_ is open and
 
 TBD: Accessibility attributes for `interesttarget`.
 
+### Security
+
+This proposal adds new capabilities in that elements can be interacted with
+using HTML where prior these required scripting (read: JS). Care has been taken
+within this proposal to ensure there are no substantially new security
+implications, and that the risk is kept low.
+
+In general, _preventing scripting is not a security barrier_ because very few
+users or websites disable scripting. That being said, we should not grant
+significant new capabilities with or without scripting.
+
+Invokers, by design, have two security affordances:
+
+- They require user interaction (invoke events and their default behaviours can
```

Comment:
is `trusted click` defined somewhere?

Reply:
This would be the first mention of it in the proposal, I think. The implementations have the check (though now you mention it the spec PR doesn't define this).

Reply:
Turns out the implementations don't have this... #945 will address this

Reply:
As per  https://github.com/openui/open-ui/issues/945#issuecomment-1815229335 this'll need rewording